ImageObs[] RGBs
ImageObs[] Depths
ScanObs[] scans
bool[] collisions
GazeboPose[] poses